pkgs/servers/routinator/default.nix

```diff
@@ -11,13 +11,13 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-JwPAwhD+Pkx8Kx24m7z/RbEvDnI2YR8dnTgAV7TMsFE=";
   };
 
+  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
   cargoSha256 = "sha256-lhSSyJxxHc0t43xoDMtr/lSVL0xZl6poPYiyYXNvKKQ=";
 
   meta = with lib; {
     description = "An RPKI Validator written in Rust";
     homepage = "https://github.com/NLnetLabs/routinator";
     license = licenses.bsd3;
     maintainers = with maintainers; [ _0x4A6F ];
-    platforms = platforms.linux;
```

Comment:
I think we should set this to platforms.all instead.

Reply:
Done.